src/library/trackset/baseplaylistfeature.cpp

```diff
@@ -599,6 +628,11 @@ void BasePlaylistFeature::bindLibraryWidget(WLibrary* libraryWidget,
     libraryWidget->registerView(m_rootViewName, edit);
 }
 
+void BasePlaylistFeature::bindSidebarWidget(WLibrarySidebar* pSidebarWidget) {
+    // store the sidebar widget pointer for later use in onRightClickChild
+    m_pSidebarWidget = pSidebarWidget;
```

Comment:
`DEBUG_ASSERT(!m_pSidebarWidget);` because initializing this member twice would be unexpected.